### DISCUSSION TITLE
Can't find previous transcripts

### DISCUSSION THREAD

**Discussion by peterenzulli** [deepgram/community#334] [General help] [answered]
Love the platform
But I can't figure out how to find a transcript from yesterday

**Comment by jjmaldonis** [comment, marked as answer]
Hey @peterenzulli, we don't store transcripts on our servers so that's why you couldn't find it. The easiest way to store a transcript is to save it to your computer when you run the transcription. You can do this in your code by writing the file to disk. If you'd rather store your transcripts in AWS S3, we have a guide that explains how to set up S3 + your Deepgram API calls so that Deepgram uploads your transcripts directly to your S3 bucket: https://developers.deepgram.com/docs/using-aws-s3-presigned-urls-with-the-deepgram-api

**Reply by peterenzulli** [reply]
thank you
